tests/python/rest_api/test_jobs.py

```diff
@@ -728,6 +732,7 @@ def _get_or_create_gt_job(self, user, task_id, frames):
         with make_api_client(user) as api_client:
             (task_jobs, _) = api_client.jobs_api.list(task_id=task_id, type="ground_truth")
             if task_jobs.results:
+                # TODO: to modify this to only get gt_job only with the mentioned frames
```

Comment:
I think it's better to add an assert here that there is no existing GT job, if specific frames are requested.